qa/suites/rgw/lifecycle/tasks/rgw_s3tests.yaml

```diff
@@ -0,0 +1,20 @@
+tasks:
+- install:
+- ceph:
+- rgw: [client.0]
+- s3tests:
+    client.0:
+      force-branch: ceph-master
```

Comment:
can you please update this to `force-branch: ceph-quincy` to match https://github.com/ceph/ceph/pull/44867?

Reply:
done

Reply:
can you please push your update? i don't see it here yet

Reply:
oops..done